Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/Views/CopyPasteHelper.cs

```diff
@@ -351,7 +356,7 @@ private static bool IsPartOfComposite(string json)
         private static SerializedProperty AddElement(SerializedProperty arrayProperty, string name, int index = -1)
         {
             var uniqueName = InputActionSerializationHelpers.FindUniqueName(arrayProperty, name);
-            if (index < 0)
+            if (index < 0 || index > arrayProperty.arraySize)
```

Comment:
Is index == arrayProperty.arraySize here?

Reply:
Not sure what you mean by this sorry?

Reply:
Sorry for the unclear question, I was wondering if index == arrayProperty.arraySize is actually a valid index in this function? Its typically not which catches my attention

Reply:
Ah, gotcha - yes it's fine, it's used for InsertArrayElementAtIndex where it's valid to add an element at the end of the array